android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java

```diff
@@ -1889,11 +1782,6 @@ private void setVerifyWalletButton() {
         String walletType = mExternalWallet.getType();
 
         switch (status) {
-            case WalletStatus.NOT_CONNECTED:
```

Comment:
Should the definition of `NOT_CONNECTED` be removed on the Android side, since it's now unused?

Reply:
we use `WalletStatus` from mojom so if we are sure of the changes on dekstop, i can remove it from mojom. i think you would need to update changes to remove grandfathered state